packages/projection/src/operators/Mappers/withHandles.ts

```diff
@@ -0,0 +1,43 @@
+import { Asset, Cardano } from '@cardano-sdk/core';
+import { FilterByPolicyIds } from './types';
+import { ProjectionOperator } from '../../types';
+import { isNotNil } from '@cardano-sdk/util';
+import { map } from 'rxjs';
+
+export interface Handle {
+  handle: string;
+  address: Cardano.PaymentAddress;
```

Comment:
Since supply tracking is done via `withMint`, the main responsibility of this operator is to track the latest owner address. Let's clarify that a little bit through naming:
```suggestion
  latestOwnerAddress: Cardano.PaymentAddress;
```